public/app/features/dashboard/components/DashboardSettings/SettingsCtrl.ts

```diff
@@ -23,6 +23,8 @@ export class SettingsCtrl {
   sections: any[];
   hasUnsavedFolderChange: boolean;
   selectors: typeof selectors.pages.Dashboard.Settings.General;
+  renderCount: number; // hack to update React when Angular changes
```

Comment:
Interesting!

Reply:
Not proud of that, `this.forceUpdate()` did not work for all cases unfortunately 😞